gapis/api/vulkan/api/device.api

```diff
@@ -59,6 +59,7 @@
   @unused ref!HalfPrecisionStorageFeatures HalfPrecisionStorageFeatures
   @unused ref!SamplerYcbcrConversionFeatures SamplerYcbcrConversionFeatures
   @unused ref!PhysicalDeviceProtectedMemoryFeatures PhysicalDeviceProtectedMemoryFeatures
+  @unused ref!PhysicalDeviceMultiviewFeatures PhysicalDeviceMultiviewFeatures
```

Comment:
Add it to the end to not break backwards-compat. I would prefer if we don't have to bump the major version before our next release. I'm OK if there are multiple `// Vulkan 1.1` sections in structs. We can re-order things after the release if we want to. I think we'll also have to decide on how we are going to handle some of that and extensions vs core stuff when I get back.

Reply:
We have decided to break it now(as we merged a fix that already breaks it).